stepped-config/README.md

```diff
@@ -0,0 +1,49 @@
+# Stepped Configuration
+
+_This is not an official Google product._
+
+This [Community Connector] shows how you can create a [stepped configuration] to
+dynamically populate a city dropdown with options based on the selected state.
+
+## Configuration
```

Comment:
I feel this should come later
1. Following the developer action sequence
2. Also this section isn't configuration for deploying the connector - rather an example for the feature.